src/components/share-structure/EditShareStructure.vue

```diff
@@ -351,21 +356,22 @@ export default class EditShareStructure extends Mixins(CurrencyLookupMixin) {
   }
 
   /** The rules applying to the Class/Series par value input field. */
-  get parValueRule (): Array<(v) => boolean | string> {
-    let rules: Array<(v) => boolean | string> = []
+  get parValueRule (): Array<VuetifyRuleFunction> {
+    let rules: Array<VuetifyRuleFunction> = []
     if (!this.hasNoParValue) {
       rules = [
-        (v: string) => (v !== '' && v !== null && v !== undefined) || 'Par value is required',
-        (v: string) => +v > 0 || 'Amount must be greater than 0',
-        (v: string) => (+v < 1)
-          ? (/^(\d+(\.\d{0,3})?|\.\d{0,3})$/.test(v) || 'Amounts less than 1 can be entered with up to 3 decimal place')
```

Comment:
I removed the second alternative because the Vuetify component doesn't allow entry of ".123". You have to enter "0.123" anyway, so the second alternative isn't needed.